tests/network/network_simplification/test_network_graph_simplificator.py

```diff
@@ -27,19 +26,27 @@ def _detailed_edge_comparison(
     graph1: nx.MultiDiGraph | nx.MultiGraph, graph2: nx.MultiDiGraph | nx.MultiGraph
 ) -> bool:
     def _dicts_comparison(
-        graph1: nx.MultiDiGraph | nx.MultiGraph, graph2: nx.MultiDiGraph | nx.MultiGraph
+        _graph1: nx.MultiDiGraph | nx.MultiGraph, _graph2: nx.MultiDiGraph | nx.MultiGraph
```

Comment:
Parameters _graph1 and _graph2 of this inner function can be omitted as _graph1 and _graph2 are know in the context of the caller.

Reply:
How to deal with the order of the arguments passed to dicts_comparison?
dicts_comparison gets once graph1 as its first argument and another time as the second. Same for graph2. So don't we need to keep _graph1 and _graph2 for dicts_comparison?

Reply:
Done!